txcache/wrappedTransaction.go

```diff
@@ -18,28 +18,33 @@ type WrappedTransaction struct {
 	ReceiverShardID uint32
 	Size            int64
 
-	PricePerUnit atomic.Uint64
+	// These fields are only set within "precomputeFields".
+	// We don't need to protect them with a mutex, since "precomputeFields" is called only once for each transaction.
+	// Additional note: "WrappedTransaction" objects are created by the Node, in dataRetriever/txpool/shardedTxPool.go.
```

Comment:
Some technical debt around this matter, we can improve the design in the future.